src/Autofac/RegistrationExtensions.cs

```diff
@@ -62,19 +62,6 @@ public static void RegisterComponent(this ContainerBuilder builder, IComponentRe
             builder.RegisterCallback(cr => cr.Register(registration));
         }
 
-        /// <summary>
```

Comment:
We can't move this out of here because that's a breaking API change.

Reply:
If you're looking for `RegisterSource<T>` you could probably pretty easily just add

```csharp
public static void RegisterSource<T>(this ContainerBuilder builder) where T : IRegistrationSource, new
{
  var source = new T();
  builder.RegisterSource(source);
}
```